### ISSUE TITLE
Move news links back to openforcefield.org/news?

### ISSUE THREAD

**Issue by jchodera** [openforcefield/openforcefield.org#310]
Any link ever published to one of the openforcefield.org news slugs (e.g. the big Parsley announcement, https://openforcefield.org/news/introducing-openforcefield-1.0/) is now broken because these have migrated to openforcefield.org/community/news/general for reasons that are likely not compelling. We should consider moving this back to openforcefield.org or at least generating URL redirects from any old slugs to their new locations so that we don't lose the history of every link any other website had to openforcefield.org.

**Comment by jchodera** [comment]
We might want to do the same with openforcefield.org/publications/... -> openforcefield.org/science/publications/...

**Comment by karmencj** [comment]
Yes, I'm aware of that and have fixed some of the broken links, but have to do it across the site and redirect old URLs to the new ones. Coming soon.
